helm/tailing-sidecar-operator/README.md

```diff
@@ -0,0 +1,37 @@
+# Tailing Sidecar Operator
+
+![Project Status](https://img.shields.io/badge/status-alpha-important?style=for-the-badge)
+
+[Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar/tree/main/operator) makes it easy to add
+[streaming sidecar containers](https://kubernetes.io/docs/concepts/cluster-administration/logging/#streaming-sidecar-container)
+to pods in your cluster by adding annotations on the pods.
+
+## Prerequisites
+
+Before installing this chart, ensure the following prerequisites are satisfied in your cluster:
+
+- [cert-manager](https://cert-manager.io/docs/installation/) is installed
+- [admission webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#prerequisites)
+  are enabled
+
+Unfortunately, due to a limitation of `cert-manager` chart, it cannot be used as a dependency and needs to be installed separately.
+See issue [jetstack/cert-manager#3062](https://github.com/jetstack/cert-manager/issues/3062) for details.
+
+## Installing
+
+Having satisfied the [Prerequisites](#prerequisites), run the following to install the chart:
+
+```sh
+helm repo add TBD TBD
+helm install tailing-sidecar-operator TBD
+```
+
+## Uninstalling
+
+```sh
+helm uninstall tailing-sidecar-operator
+```
+
+## Configuration
+
+TBD
```

Comment:
In my opinion configuration options are now known -all options from values.yaml. Maybe it will be good at least mention this file here.

Reply:
Done in https://github.com/SumoLogic/tailing-sidecar/pull/82/commits/f95b147081d5ad83a87c5462fa4c4d3c835140f1